manifests/plugin/salt.pp

```diff
@@ -42,4 +45,12 @@
     enabled   => $enabled,
     listen_on => $listen_on,
   }
+
+  file {"${foreman_proxy::etc}/salt/master.d/foreman.conf":
```

Comment:
Is this directory guaranteed to exist? I think this certainly needs some dependency which guarantees the parent directory exists.

Reply:
@ekohl How can I implemented this? Can I easily add the directory creation as dependency to this `file`-part?